cli/proc_state.rs

```diff
@@ -414,6 +424,21 @@ impl ProcState {
           check_js,
         )
         .unwrap()?;
+      graph_data.npm_package_reqs()
+    };
+
+    if !npm_package_references.is_empty() {
+      self
+        .npm_resolver
+        .add_package_reqs(npm_package_references)
+        .await?;
+      self.npm_resolver.cache_packages().await?;
+
+      // add the builtin node modules to the graph data
+      let node_std_graph = self
+        .create_graph(vec![(compat::MODULE_ALL_URL.clone(), ModuleKind::Esm)])
```

Comment:
This is very very slow and we should remove it somehow.

Reply:
https://github.com/denoland/deno_graph/pull/173 will unblock this, IMO not a blocker to land it now and improve in a followup

Reply:
The slowness is quite bad... it takes almost 2 seconds to load a very basic npm dependency that's already been downloaded.

Reply:
David is working on fixing this, but it requires to land numerous PRs across various repos. Let's land this PR as is and we'll fix it in a follow up PR.